src/profile-logic/process-profile.js

```diff
@@ -533,6 +534,22 @@ function _processMarkers(geckoMarkers: GeckoMarkerStruct): MarkersTable {
                 throw new Error('Unknown GCMajor status');
             }
           }
+          case 'tracing': {
+            const newData = Object.assign({}, m);
+            delete newData.stack;
+            if ('stack' in m && m.stack && m.stack.samples.data.length > 0) {
+              const syncProfile = m.stack;
```

Comment:
what's this name `syncProfile`? Why not `stack` ?

Reply:
Renamed to stack.